samples/samples/backup_sample.py

```diff
@@ -380,6 +380,83 @@ def create_database_with_version_retention_period(instance_id, database_id, rete
 
 # [END spanner_create_database_with_version_retention_period]
 
+# [START spanner_copy_backup]
+def copy_backup(instance_id, database_id, backup_id, source_backup_id):
+    """Copied a backup for a database."""
+    spanner_client = spanner.Client()
+    instance = spanner_client.instance(instance_id)
+    database = instance.database(database_id)
+
+    # Create a source backup and wait for create backup operation to complete.
+    expire_time = datetime.utcnow() + timedelta(days=14)
+    source_backup = instance.backup(source_backup_id, database=database, expire_time=expire_time)
+    operation = source_backup.create()
```

Comment:
Rather than creating a new backup, could we instead use an existing backup?

```
spanner_client = spanner.Client()
instance = spanner_client.instance(instance_id)
database = instance.database(database_id)
source_backup = instance.backup(source_backup_id)
source_backup.reload()  # Fetch backup details from backend.
```

Creating backups is expensive so it's preferred to reuse existing ones where possible.